db/migrate/20231108210613_change_associations_duration_rates.rb

```diff
@@ -0,0 +1,19 @@
+class ChangeAssociationsDurationRates < ActiveRecord::Migration[7.0]
+  def up
+    DurationRate.destroy_all
```

Comment:
Can you use either SQL or define `DurationRate` in the migration file?  Just to be [safe](https://jakeyesbeck.com/2021/04/10/avoid-models-in-migrations/)